pkg/apis/networking/types.go

```diff
@@ -138,10 +138,19 @@ type NetworkPolicyPort struct {
 	// +optional
 	Protocol *api.Protocol
 
-	// The port on the given protocol. This can either be a numerical or named port on
-	// a pod. If this field is not provided, this matches all port names and numbers.
+	// The port on the given protocol. This can either be a numerical or named
+	// port on a pod. If this field is not provided, this matches all port names and
+	// numbers, whether an endPort is defined or not.
+	// If present, only traffic on the specified protocol AND port
 	// +optional
 	Port *intstr.IntOrString
+
+	// EndPort defines the end of the port range, being the end included within the
+	// range
+	// Example:
+	//    endPort: 12345
```

Comment:
This example is probably not that useful without context of the other fields. I would just remove it.